.prettierignore

```diff
@@ -2,4 +2,9 @@
 src/emoji/__tests__/data-test.js
 
 # We're not allowing Prettier to touch some of our vendored code.
-src/third/redux-persist
\ No newline at end of file
+src/third/redux-persist
+
+# We'd like to keep the original formatting for some Flow libdefs from
+# FlowTyped, to make it easier to eventually make a PR back to
+# FlowTyped.
+flow-typed/@react-navigation
```

Comment:
nit: last line should end with a newline, like other lines

(Git will display this like so:
```diff
+flow-typed/@react-navigation
\ No newline at end of file
```
)

Reply:
Ah, yeah, thanks for the catch. This is one of those things that I know we're supposed to do, but that I don't usually think about—probably because our auto-formatting takes care of it in most files.